prediction_market_agent/agents/safe_guard_agent/safe_api_models/detailed_transaction_info.py

```diff
@@ -27,81 +29,88 @@ class TransferInfo(BaseModel):
     value: str
 
 
+class AddressInfo(BaseModel):
```

Comment:
For Pydantic models, they are mostly auto-generated based on transactions. After the option to allow validation of non-owner Safes, I used Martin's safe transactions to auto-fix these models. I think we don't have to worry about them too much atm as we have other issues for that.